beacon-chain/cache/committee.go

```diff
@@ -23,6 +23,11 @@ const (
 	// maxCommitteesCacheSize defines the max number of shuffled committees on per randao basis can cache.
 	// Due to reorgs and long finality, it's good to keep the old cache around for quickly switch over.
 	maxCommitteesCacheSize = int(32)
+
+	// delay parameters for checking in progress requests.
+	minDelay    = float64(10)        // 10 nanoseconds
+	maxDelay    = float64(100000000) // 0.1 second
+	delayFactor = 1.1
```

Comment:
These configs were shared between attestation data cache and committee cache. Now that attestation data cache was deleted, we are moving the configs here